src/wix/WixToolset.Core/Compiler_Package.cs

```diff
@@ -242,9 +242,15 @@ private void ParsePackageElement(XElement node)
                         case "Component":
```

Comment:
Is this parsing `Component` in `Package` element? Should we change the `ComplexReferenceParentType.Unknown` to the new `Product` option?